CHANGELOG.asciidoc

```diff
@@ -3,6 +3,55 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.11.2]]
+=== Beats version 8.11.2
+https://github.com/elastic/beats/compare/v8.11.1\...v8.11.2[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+- Elastic-agent-autodiscover library updated to version 0.6.4, disabling metadata for deployment and cronjob. Pods that will be created from deployments or cronjobs will not have the extra metadata field for kubernetes.deployment or kubernetes.cronjob, respectively. {pull}36877[36877]
```

Comment:
I think this one was released already as well.

https://github.com/elastic/beats/blob/34f53905653d671b679f64b046b8bc153095b702/CHANGELOG.asciidoc?plain=1#L25